### PR TITLE
Layer5 and Meshery branding update

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,8 @@ We create and steward initiatives that press on the service mesh-centric envelop
 <h2><a href="https://layer5.io/meshery">Meshery</a></h2>
 <a href="https://meshery.io"><img src=".github/assets/images/meshery/meshery-logo.svg"
 style="margin:10px;" width="125px"
-alt="Meshery - Service Mesh Management Plane" align="left" /></a>
-<a href="https://meshery.io">Meshery</a> is a multi-service mesh management plane offering lifecycle management of more types of service meshes than any other tool available today.
-Meshery facilitates adopting, configuring, operating and managing performance of different service meshes and incorporates the collection and display of metrics from applications running on top of any service mesh.
+alt="Meshery - Cloud NativeManagement Plane" align="left" /></a>
```
Comment (leecalcote) [inline README.md:28]:
```suggestion
alt="Meshery - Cloud Native Management Plane" align="left" /></a>
```

File: src/collections/news/2021/2021-10-13-cncf-adopts-meshery/index.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "CNCF Adopts Meshery, the Service Mesh Management Plane"
+title: "CNCF Adopts Meshery, the cloud native management plane"
```
Comment (leecalcote) [inline src/collections/news/2021/2021-10-13-cncf-adopts-meshery/index.mdx:2]:
```suggestion
title: "CNCF Adopts Meshery, the Service Mesh Management Plane"
```

File: src/collections/news/2020/2020-10-29-kubernetes-evolves-into-an-enterprise-platform.md/index.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title:  "Meshery Provides the Service Mesh Management Plane and Kubernetes Evolves into an Enterprise Platform"
+title:  "Meshery Provides the cloud native management plane and Kubernetes Evolves into an Enterprise Platform"
```
Comment (leecalcote) [inline src/collections/news/2020/2020-10-29-kubernetes-evolves-into-an-enterprise-platform.md/index.mdx:2]:
```suggestion
title:  "Meshery Provides the Service Mesh Management Plane and Kubernetes Evolves into an Enterprise Platform"
```

File: src/collections/news/2021/2021-10-13-cncf-adopts-meshery/index.mdx [hunk 4]
```diff
@@ -20,9 +20,9 @@ import { Link } from "gatsby";
 
 <NewsWrapper>
 
-LOS ANGELES, CA, Oct. 13, 2021 - ServiceMeshCon/KubeCon + CloudNativeCon:  Layer5 today announced Meshery, the service mesh management plane, has been adopted by the CNCF. Offering lifecycle, configuration, and performance management, <Link to="/service-mesh-management/meshery">Meshery</Link> enables the confident operation of any service mesh and their workloads. Built for the world of many service meshes, Meshery interoperates with ten different service meshes embracing their differentiated features. 
+LOS ANGELES, CA, Oct. 13, 2021 - ServiceMeshCon/KubeCon + CloudNativeCon:  Layer5 today announced Meshery, the cloud native management plane, has been adopted by the CNCF. Offering lifecycle, configuration, and performance management, <Link to="/service-mesh-management/meshery">Meshery</Link> enables the confident operation of any service mesh and their workloads. Built for the world of many service meshes, Meshery interoperates with ten different service meshes embracing their differentiated features. 
 
-<h3>The Service Mesh Management Plane </h3>
+<h3>the cloud native management plane </h3>
```
Comment (leecalcote) [inline src/collections/news/2021/2021-10-13-cncf-adopts-meshery/index.mdx:25]:
```suggestion
<h3>The Service Mesh Management Plane</h3>
```

File: static/mail/kubecon-na-2021-recap.html [hunk 5]
```diff
@@ -578,7 +578,7 @@
                         
                         <td valign="top" class="mcnTextContent" style="padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;">
                         
-                            <h2 style="text-align: center;">CNCF Adopts Meshery, the Service Mesh Management Plane</h2>
+                            <h2 style="text-align: center;">CNCF Adopts Meshery, the cloud native management plane</h2>
```
Comment (leecalcote) [inline static/mail/kubecon-na-2021-recap.html:581]:
```suggestion
                            <h2 style="text-align: center;">CNCF Adopts Meshery, the Service Mesh Management Plane</h2>
```

**Comment by l5io** [comment]
🚀 Preview for commit 03509be5dab220a38f5e2e4fd1586994f749bc10 at: https://628695a4334bfc1c8b0a3add--layer5ng.netlify.app

**Pull request merged**